src/Tizen.NUI/src/public/Theme/ThemeManager.cs

```diff
@@ -161,6 +161,25 @@ public static void ApplyTheme(Theme theme)
                 newTheme.Id = "NONAME";
             }
 
+            if (string.IsNullOrEmpty(newTheme.SmallBrokenImageUrl))
+            {
+                newTheme.SmallBrokenImageUrl = "";
```

Comment:
```C#
var theme = new Theme();
Console.WriteLine("{0}", theme.SmallBrokenImageUrl); // -> null

ThemeManager.ApplyTheme(theme);
Console.WriteLine("{0}", theme.SmallBrokenImageUrl); // -> ""
```
I think it'd be better not to change given value from the user.

For the null check, how about this?
```C#
StyleManager.Instance.SetBrokenImageUrl(StyleManager.BrokenImageType.Small, newTheme.SmallBrokenImageUrl ?? "");
```

Reply:
i think it'd be better!  thank you :)